ux.symfony.com/assets/demos/live-memory.js

```diff
@@ -0,0 +1,6 @@
+import { startStimulusApp } from '@symfony/stimulus-bundle';
+import Clipboard from 'stimulus-clipboard';
+
+const app = startStimulusApp();
+
+app.register('clipboard', Clipboard);
```

Comment:
We should import `./bootstrap.js` instead.

Reply:
As i said, the code was 100% independant waiting to see how we integrate it :)